docs/writing-rules/pattern-syntax.mdx

```diff
@@ -364,6 +364,25 @@ You can also use `/$X/` and `:$X` to respectively match
 any regular expressions or atoms (in languages that support
 those constructs, e.g., Ruby).
 
+:::info
+Because literal metavariables bind to strings that may not be valid code, if you want to match them in more detail with a [`metavariable-pattern`](/writing-rules/rule-syntax/#metavariable-pattern), you'll need to [specify `generic` language](/writing-rules/rule-syntax#metavariable-pattern-with-nested-language) inside the `metavariable-pattern`. For example:
```

Comment:
```suggestion
Because literal metavariables bind to strings that may not be valid code, if you want to match them in more detail with a [`metavariable-pattern`](/writing-rules/rule-syntax/#metavariable-pattern), you must [specify `generic` language](/writing-rules/rule-syntax#metavariable-pattern-with-nested-language) inside the `metavariable-pattern`. For example:
```

Reply:
Open to this change, but I'm interested in your reasoning -- does our brand voice lean more formal than I'm used to?

Reply:
Hi, it's totally reasonable to ask that question! We use the google style guide. 

`Will` is not banned as much as it is in google, but it's still avoided where possible: 

- https://developers.google.com/style/word-list#will
- https://developers.google.com/style/tense

`You (will) need to` is longer than `You must` and where possible we try to shorten and be prescriptive, especially for instructions where the user _has_ to do the thing, there's no other option.